includes/emicdora.inc

```diff
@@ -26,10 +27,31 @@ function collationtools_get_edited_collation($document_id) {
       'version2' => $_GET['version2'],
       'diff_kind' => $_GET['diff_kind'],
   )));
-  $contents = file_get_contents($url);
-  $spans = preg_split( '#</span>#', $contents, -1, PREG_SPLIT_DELIM_CAPTURE | PREG_SPLIT_NO_EMPTY);
-  $fixed_content = collationtools_fix_calliope($spans, $_GET['diff_kind']);
-
+  $raw_contents = file_get_contents($url);
+  $contents = str_replace($br_token, '<br />', $raw_contents);
+  $dom = new DOMDocument();
+  $dom->loadHTML($contents);
+  $bodies = $dom->getElementsByTagName('body');
+  $body = $bodies->item(0);
+  // Get rid of comment nodes.
+  foreach ($dom->childNodes as $node) {
```

Comment:
Not necessary - there's only ever one comment per MVD - this is just to clear the unused css out of the input.